app/controllers/sessions_controller.rb

```diff
@@ -6,7 +6,7 @@ def create
     user = find_user
     if user&.authenticate(params[:session][:password])
```

Comment:
this method has more than two responsibilities, please look into it. 

{Remind to talk on this one too.}